docs/release_notes.rst

```diff
@@ -14,6 +14,7 @@ New Features
 Bug Fixes
 
 * Update convention support documentation (#386, #393)
+* Detect inner asynchronous functions (#467)
```

Comment:
One small comment - can you update this to say that it relates to D202?

Reply:
Sure, done!